pkg/build/api/v1/types.go

```diff
@@ -402,6 +408,10 @@ type SourceBuildStrategy struct {
 	ForcePull bool `json:"forcePull,omitempty" description:"forces the source build to pull the image if true"`
 }
 
+// ExternalBuildStrategy defines input parameters specific to an External build.
+type ExternalBuildStrategy struct {
```

Comment:
(where JobURL is the url of the job definition, not the specific instance of the job that has been launched, since that information goes on the Build object)